src/main/java/seedu/gamebook/logic/commands/AddCommand.java

```diff
@@ -56,7 +56,7 @@
     public static final String MESSAGE_USAGE = COMMAND_FORMAT + "\n" + COMMAND_SPECIFICATION;
     public static final String MESSAGE_SUCCESS = "New game added: \n%1$s\n%2$s";
     public static final String MESSAGE_DUPLICATE_GAME_ENTRY = "Alert: A game entry with the same "
-        + "game type, date and time already exists.";
+        + "game type and date/datetime already exists.";
```

Comment:
Perhaps we could clarify what we mean by date and datetime in the UG

Reply:
I think we should clarify this in the UG otherwise having an alert that is too long might be hard to understand.